content/docs/next-release/_index.md

```diff
@@ -30,7 +30,8 @@ Uber published a blog post, [Evolving Distributed Tracing at Uber](https://eng.u
 
   * [OpenTracing](http://opentracing.io/)-inspired data model
   * Uses consistent upfront sampling with individual per service/endpoint probabilities
-  * Multiple storage backends: Cassandra, Elasticsearch, memory.
+  * Multiple storage inbuilt backends: Cassandra, Elasticsearch, in-memory
+  * Community supported external storage backend via gRPC plugin: [PostgreSQL with Promscale](https://github.com/timescale/promscale#promscale-for-jaeger-and-opentelemetry), [ClickHouse](https://github.com/jaegertracing/jaeger-clickhouse)
```

Comment:
I am not sure what is the best way to bring this here. If you think this is the best way, go for it.